gpustack/policies/candidate_selectors/sglang_resource_fit_selector.py

```diff
@@ -126,18 +143,85 @@ def _get_nnodes(self) -> int:
 
         return 1
 
-    def _validate_arguments(self):
+    def _get_max_tp_size(self) -> int:
+        cmp_list = [self._tp_size, self._select_tp_size]
+        if self._model.gpu_selector and self._model.gpu_selector.gpus_per_replica:
+            cmp_list.append(self._model.gpu_selector.gpus_per_replica)
+        return max(cmp_list)
+
+    async def _get_min_gpu_sum(self, workers: List[Worker]) -> int:
+        """
```

Comment:
See if this function can be reused to simplify the current function
https://github.com/gpustack/gpustack/blob/ea516431f108953913c3d96772336e190d1c9504/gpustack/policies/candidate_selectors/base_candidate_selector.py#L272